example/conformity/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "autobotai_conformity_integration" "conformity" {
```

Comment:
For all the integrations, create a folder inside example named "integrations" and put them there.